pravega-sensor-collector/src/main/java/io/pravega/sensor/collector/simple/PersistentQueue.java

```diff
@@ -68,14 +68,11 @@ public static Connection createDatabase(String fileName) {
         try {
             final Connection connection = DriverManager.getConnection("jdbc:sqlite:" + fileName);
             try (final Statement statement = connection.createStatement()) {
-                // Use SQLite exclusive locking mode to ensure that another process or device driver instance is not using this database.
-                //statement.execute("PRAGMA locking_mode = EXCLUSIVE");
-                statement.execute(
-                        "create table if not exists Queue (" +
-                                "id integer primary key autoincrement, " +
-                                "bytes blob not null, " +
-                                "routingKey string not null, " +
-                                "timestamp integer not null)");
+                // Use SQLite exclusive locking mode to ensure that another process or device
+                // driver instance is not using this database.
+                // statement.execute("PRAGMA locking_mode = EXCLUSIVE");
+                statement.execute("create table if not exists Queue (" + "id integer primary key autoincrement, "
```

Comment:
The autoformatting of this line left the `+` characters and makes it a bit hard to read. Either remove the `+` characters or under the autoformatting for this statement.

Reply:
Changed the formatting